### ISSUE TITLE
Generating docs broken on error (undefined method ReflectionProperty::getDefaultValueAsString())

### ISSUE THREAD

**Issue by devgkz** [ApiGen/ApiGen#964]
Error message:
```
[ApiGen\Exception\Templating\FailedRenderFileException]
... 
Call to undefined method
  Roave\BetterReflection\Reflection\ReflectionProperty::getDefaultValueAsString()
```

**Comment by TomasVotruba** [comment]
Resolved by #966

**Comment by metalcamp** [comment]
Looks like I got stuck on same error. When is next release planned? Or is there a way to build a package myself? 

Sorry, I'm not too familiar with PHP packages.
